pkg/pgx_conn/pgx_conn.go

```diff
@@ -0,0 +1,67 @@
+package pgxconn
```

Comment:
Is there a good reason why the package is named `pgxconn` and the directory `pgx_conn`? If not, I would prefer them called the same.